### PR TITLE
[BACKLOG-5384, TEST, CLEANUP] - Increase unit test coverage

### PR DESCRIPTION
**Pull request by plagoa**
- Increased unit test coverage;
- Added the Mockito dependency;
- Applied CheckStyle to existing classes;

### CODE REVIEW

**Comment by plagoa** [comment]
@diogofscmariano @pamval please review

File: test-src/org/pentaho/reporting/platform/plugin/output/PageableHTMLOutputTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.pentaho.reporting.platform.plugin.output;
```
Comment (pamval) [inline test-src/org/pentaho/reporting/platform/plugin/output/PageableHTMLOutputTest.java]: license header missing

File: test-src/org/pentaho/reporting/platform/plugin/repository/ReportContentLocationTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+package org.pentaho.reporting.platform.plugin.repository;
+
+import junit.framework.TestCase;
+import org.pentaho.platform.api.repository2.unified.RepositoryFile;
+
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by webdetails on 08/10/2015.
+ */
```
Comment (pamval) [inline test-src/org/pentaho/reporting/platform/plugin/repository/ReportContentLocationTest.java]: Remove this comment, please.

File: test-src/org/pentaho/reporting/platform/plugin/repository/ReportContentLocationTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2002-2013 Pentaho Corporation..  All rights reserved.
```
Comment (pamval) [inline test-src/org/pentaho/reporting/platform/plugin/repository/ReportContentLocationTest.java]: Update copyright year to current year in the new files you're creating

**Comment by plagoa** [comment]
@pamval removed the "created by" comments and updated the copyright year.

**Comment by pamval** [comment]
but you did not add the license header to PageableHTMLOutputTest

**Comment by plagoa** [comment]
I believe I did: https://github.com/plagoa/pentaho-platform-plugin-reporting/blob/ecc882c1dd12370a559885e3bf8ceca3f4227a26/test-src/org/pentaho/reporting/platform/plugin/output/PageableHTMLOutputTest.java

**Comment by pamval** [comment]
bah, github did not update

**Pull request merged**